portal-web/docroot/html/taglib/ui/panel/lexicon/start.jsp

```diff
@@ -33,12 +33,12 @@ if (persistState) {
 }
 %>
 
-<div class="panel panel-secondary <%= cssClass %>" id="<%= id %>">
-	<div class="panel-heading" id="<%= id %>Header" role="tab">
-		<div class="h4 panel-title">
-			<c:choose>
-				<c:when test="<%= collapsible %>">
-					<a aria-controls="<%= id %>Content" aria-expanded="<%= !collapsed %>" class="collapse-icon collapse-icon-middle <%= collapsed ? "collapsed" : StringPool.BLANK %>" data-parent="#<%= id %>" data-toggle="liferay-collapse" href="#<%= id %>Content" role="button">
+<div class="panel <%= cssClass %>" id="<%= id %>">
+	<div class="panel panel-unstyled" id="<%= id %>Header" role="tab">
```

Comment:
We should remove line 37 and move `panel-unstyled` to line 36 and `id` and `role` to the anchor element. It should output `panel-unstyled` if a `cssClass` isn't defined, otherwise use the `cssClass` that is defined.

It also might be a good opportunity to add `c-inner` so we don't get focus outline on click.

Otherwise it LGTM.

Reply:
I was trying to be the least intrusive with this old taglib, but your changes seem fine to me! I'm going to add them.
By the way, this actually .. is not a tab but it has a role tab ...

Reply:
`cssClass` looks like is always defined with `lfr-panel lfr-panel-extended` so by now I'm going to keep `panel-unstyled` for all panels (77 uses in portal)